### DISCUSSION TITLE
Which pipe?

### DISCUSSION THREAD

**Discussion by jmgirard** [PittMethods/appliedml#24] [General]
Do we want to use the magrittr pipe `%>%` or the native R pipe `|>` in this course? The latter requires no packages but does require R 4.1.0+ and a setting to be changed in RStudio to use the keyboard shortcut. The former requires magrittr package to be installed and loaded (but is included in tidyverse).

**Comment by ShirleyBWang** [comment]
I've been using %>% in all my slides to date but I'm flexible! No strong feelings either way

**Reply by jmgirard** [reply]
I feel like `|>` will take over in the next year or two, but for now both seem acceptable.

**Reply by jmgirard** [reply]
I taught `|>` in my intro to R course, but the R4DS and TMWR books both use `%>%`.

**Reply by ShirleyBWang** [reply]
You're probably more well-versed on tidyverse trends then I am - as an infrequent user I wasn't even really aware of |>!

**Reply by ShirleyBWang** [reply]
Hmm, for consistency's sake with TMWR maybe we should stick with `%>%` ? We can note that `|>` does the same though. Sort of like `<-` and `=`; mostly personal preference.

**Reply by jmgirard** [reply]
Well, the difference is that `%>%` is tidyverse only and `|>` is just R now.

**Reply by jmgirard** [reply]
But sure, let's stick with the magrittr pipe for now.

**Reply by ShirleyBWang** [reply]
Sounds good, either is honestly fine with me - can easily find/replace `%>%` to `|>` if needed!
